package.json

```diff
@@ -28,6 +28,7 @@
     "@types/node": "^16.0.0",
     "@types/ws": "^7.4.0",
     "@typescript-eslint/parser": "^4.0.0",
+    "eslint": "^7.31.0",
```

Comment:
why?

Reply:
`npm test` was complaining this missing dep (I don't have `eslint` installed globally)

Reply:
Something got messed up in your installation.

Reply:
turns out it was related to [this](https://github.com/standard/standard/issues/1439#issuecomment-562983862) .